argus/recipes/cloud/windows.py

```diff
@@ -265,22 +275,33 @@ def _wait_cbinit_finalization(self, searched_paths=None):
         LOG.info("Waiting for the finalization of CloudbaseInit execution...")
 
         # Check if the service actually started.
-        test_cmd = 'powershell Test-Path {}'
+        test_cmd = 'Test-Path {}'
         check_cmds = [test_cmd.format(introspection.escape_path(path))
                       for path in searched_paths or []]
         for check_cmd in check_cmds:
             self._execute_until_condition(
                 check_cmd,
                 lambda out: out.strip() == 'True',
-                count=COUNT, delay=DELAY)
+                count=COUNT, delay=DELAY, command_type=util.POWERSHELL)
 
         # Check if the service finished
-        wait_cmd = ('powershell (Get-Service "| where -Property Name '
-                    '-match cloudbase-init").Status')
+        wait_cmd = ('(Get-Service | where -Property Name '
+                    '-match cloudbase-init).Status')
+        try:
+            self._execute_until_condition(
+                wait_cmd,
+                lambda out: out.strip() == 'Stopped',
+                count=COUNT, delay=DELAY, command_type=util.POWERSHELL)
+        except exceptions.ArgusCLIError as err:
+            # This error ocures when you execute the first command when you boot 
+            # the machine
+            LOG.debug(("We got and ArgusCLIError and it is ignored"
+                      " with the output : {}").format(err.message))
```

Comment:
Use `%` formatting in logging functions but pass the `%` parameters as arguments (logging-format-interpolation).